docs-site/app/global.css

```
@@ -277,3 +476,1774 @@
+  z-index: 80;
+  height: 2px;
+  pointer-events: none;
+  background: linear-gradient(90deg, var(--brand-cisco), #65c7ea);
+  will-change: transform;
+}
+
+/* Connector identity --------------------------------------------------- */
+
+.connector-brand-mark {
+  --connector-brand: var(--brand-cisco);
+  display: inline-flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)